lib/launch-fastify.js

```diff
@@ -35,46 +35,51 @@ const lc39Options = {
   disableRequestLogging: true,
 }
 
-function importModule(filePath) {
-  // disable the global require rule for importing the right module from the user given file
-  // eslint-disable-next-line global-require
-  const importedModule = require(absolutePath(filePath))
-
+function checkAsyncFunction(importedModule) {
   if (importedModule.constructor.name !== 'AsyncFunction') {
     // we expect that all services will export an async function
     throw new Error('The module doesn’t export an async function')
   }
+}
 
+function checkParamsLength(importedModule) {
   if (importedModule.length > 2) {
     // we expect that all the services will export a function that will accept at max two arguments:
     // fastify and options
     throw new Error('Init function should contain max 2 arguments')
   }
+}
+
+function importModule(filePath) {
+  // disable the global require rule for importing the right module from the user given file
+  // eslint-disable-next-line global-require
+  const importedModule = require(absolutePath(filePath))
+  checkAsyncFunction(importedModule)
+  checkParamsLength(importedModule)
   return importedModule
 }
 
+function isRegexValidated(value, regex) {
+  return !value || regex.test(value)
```

Comment:
If I'm correct, this change introduces a breaking change, in fact prior this change the `validatePrefix` function provided with a falsy `prefix` would simply return undefined, now it will throw an error instead.

I think this is not emerged because there may be some missing tests, can you please verify this change with a new test?

Reply:
Hi @fredmaggiowski,
I've improved the tests of launch-fastify with also good prefixes example, but I can't reach the use case that you're thinking about.

Reply:
I've seen the test and it sould legit, perfect; thanks!